### ISSUE TITLE
Support for NET 7

### ISSUE THREAD

**Issue by JonPSmith** [Giorgi/EntityFramework.Exceptions#54]
Hi @Giorgi,

Can you create a version for EF Core 7. Thanks.

**Comment by Giorgi** [comment]
Is the current version incompatible with EF Core 7?

**Comment by JonPSmith** [comment]
Hi @Giorgi,

It did have a problem, which is why I opened the issue, but later it worked! That says the code works. 

I have found that if I can use a netstandard as the TargetFramework, then you don't do anything when a new .NET version comes out, unless EF Core changes something you use. 

My EfCore.TestSupport library has to have netstandard2.1, net6.0 and net7.0 because a small thing had changed in EF Core, but my EfCore.GenericBizRunner library uses netstandard2.0 because it was using basic EF Core features.

So, you don't need to create a .NET 7 as it works, but you might want to think how to cover new .NET releases  - see my [Twitter thread](https://twitter.com/thereformedprog/status/159471426143951258) about me updating my libraries to support .NET 7 (and beyond).
